lblod/frontend-reglementaire-bijlage
# Add snippet-placeholder node and insert menu entry

## Overview
Add the snippet placeholder insert control to RB.

##### connected issues and PRs:
[binnenland.atlassian.net/browse/GN-4782](https://binnenland.atlassian.net/browse/GN-4782)
Uses: https://github.com/lblod/ember-rdfa-editor/pull/1188
And: https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/419

### Setup
N/A

### How to test/reproduce
Should be able to insert a placeholder into a template, choosing the snippet lists, then insert a snippet in place of it. Also interesting to test alongside the equivalent GN branch.

### Challenges/uncertainties
N/A


### Checks PR readiness
